contrib/ansible/roles/etcd/tasks/main.yml

```diff
@@ -8,6 +8,11 @@
     - restart etcd
   when: not is_atomic
 
+- name: Clean etcd data directory
+  shell: rm -rf {{ etcd_data_dir }}/*
+  notify:
+    - restart etcd
```

Comment:
@yichengq do you have any recommendations here?  Basically we would like to preserve the data, but nuke the membership list to take a new config.